# The action goal for creating a rectangle
string turtle_name
int32 move_distance
int32 move_speed
float32 degrees
bool isForward
bool isClockwise
int32 speed_rotate
---
# Define result's message
---
# Define feedback's message

